Kiwix.xcodeproj/project.pbxproj

```diff
@@ -655,7 +664,7 @@
 				};
 			};
 			buildConfigurationList = 97A2AB831C1B80FF00052E74 /* Build configuration list for PBXProject "Kiwix" */;
-			compatibilityVersion = "Xcode 13.0";
+			compatibilityVersion = "Xcode 15.0";
```

Comment:
This is not something we can changed just like this usually. @rgaudin Is that OK?

Reply:
No problem, reverted to version 13.